docs/4-functional-programming/homework/src/fp/maybe.ts

```diff
@@ -27,7 +27,9 @@ export const none: Readonly<None> = {
 /**
  * Create a Maybe instance form the value. If value(T) is nullable(null or undefined), returns None, otherwise it returns Some<T>
  */
-export const fromNullable = <T>(value: T) => ();
+export const fromNullable = <T>(value: T) => {
```

Comment:
we can simplify it like `export const fromNullable = <T>(value: T) => value ? some(value) : none`